ngraph/test/runtime/interpreter/int_executable.hpp

```diff
@@ -892,6 +893,16 @@ class INTERPRETER_BACKEND_API ngraph::runtime::interpreter::INTExecutable : publ
                                              pbox->get_attrs());
             break;
         }
+        case OP_TYPEID::ReorgYolo_v0:
```

Comment:
As I understand all method which have evaluate are calculated during Interpreter execution (look here: https://github.com/mbencer/openvino/blob/926be8356887567e1f58a7e55bc34a01cc660ed1/ngraph/test/runtime/interpreter/int_executable.cpp#L205). Why is it called directly?

Reply:
It was recommended to not create evaluate method by default, so ReorgYolo does't have `evaluate`.

Reply:
This op won't be used in Constant Folding, hence it should be called via int_executable (not via evaluate()) to not unnecessarily increase ngraph binary size.

Reply:
right, I miss it ;)